### ISSUE TITLE
./runeson/Plea4vis-v1.md

### ISSUE THREAD

**Issue by timm** [ds4se/chapters#73]
After review, relabel to 'reviewTwo'. After second review, relabel to 'EditorsComment'.

**Comment by tostrand** [comment]
## Title of chapter

MERE NUMBERS AREN'T ENOUGH: A PLEA FOR VISUALIZATION

## URL to the chapter

https://github.com/ds4se/chapters/blob/master/runeson/Plea4vis-v1.md

## Message?

Graphical visualization of data pertaining to software projects is an essential method for providing insight and understanding of the status of a project. 

## Accessible?

Yes, the chapter is accessible. It's a nice concept, I'm in complete agreement on the value of visualizing data, when it's done well.  The two examples are classic software engineering tasks that can be helped by appropriate visualization of relevant data.  

## Size?

The chapter could be lengthened, to provide better explanations of the examples. 
Specific suggestions: 

FEATURE SCOPING

"scope out" sounds strange (to my American ear).  It would be much easier to understand if "exclude" were used.  For example:

Replace "... which to scope out from the current project." 
with "... which to exclude from the current project. " 

scoped out -> excluded

I don't understand the phrase "basic average share of descoped features". 

The next sentence isn't really a sentence.  What didn't happen "until they presented the feature survival chart" ??  

Presumably Figure X means figure 1.
But the figure itself is confusing, and the text seems to be confused also.  The text seems to have the x and y-axes interchanged.  Aren't the features on the y-axis, and the milestones on the x-axis? 

Instead of "nuances of green", it would be better to say "shades of green".  
The paragraph just above the figure mentions the marked decision points numbered 1  through 5, but doesn't explain anything about what the decisions were, or how the feature scoping affected the decisions.  Even in this short overview article, at least a couple of sentences about the actual use of the chart would be helpful. 

REGRESSION TESTING

Presumably Figure Y means figure 2.

The picture looks colorful and nice, but without an explanation, I can only guess what the different colors mean. Does the first row mean that Unit Testing was done intensely on about 15 features in V 3.0, intensely on 2, and less intensely on 12 features in 3.2, and not at all in 3.3?  

## Gotta Mantra?

The title is acceptable. 

## Best Points

The examples are good, but they need to be better explained.

**Comment by devanbu** [comment]
## Title of chapter

MERE NUMBERS AREN'T ENOUGH: A PLEA FOR VISUALIZATION

## URL to the chapter

https://github.com/ds4se/chapters/blob/master/runeson/Plea4vis-v1.md

## Message?

The author is making a case that visualization adds value beyond quantitative and qualitative analysis. 

## Accessible?

Generally, Yes, although the specifics are hard to follow. 

The two visualizations presented are both difficult to follow. I suggest picking one or two specific
examples (features in teh first case, and tests/components in the second) and explaining in a bit more detail what the colors mean, and why this particular visualization adds value. 

## Size?

Is the chapter the right length?
Yes
Should anything missing be added?

See above. 
?

## Gotta Mantra?

N/A

## Best Points

Visualizations are pretty, even if hard to understand :-)

**Comment by tzimmermsr** [comment]
This sentence does not parse (what happened after they presented the feature survival chart?): "However, not until they presented the feature survival chart, visualizing features and their status change in an x-y-diagram, see figure X."

The axes are swapped in the description:
- The features are listed on the x axis => y axis
- and time and milestones are marked on the y axis. => x axis

Figure X => Figure 1

It would be great to list some of the major decisions ("able to connect major decisions in the project to the scope, marked with numbers"

Figure Y => Figure 2

Spend a few more sentences on explaining Figure 2, e.g., what does the layout of the dots mean? Talk about how the visualization supports multiple versions, and test levels.

New paragraph before "We evaluated the tool"

**Comment by tzimmermsr** [comment]
@pruneson46 Please prepare a new version of your paper by January 13 taking the reviewers' feedback into account. A theme among all reviews is that the visualizations need more explanation.

It's OK if those explanations make the chapter slightly longer (or go beyond the word limit).

**Comment by tzimmermsr** [comment]
Thanks @pruneson46 for the revision. The chapter is good to go from my side. --Tom
